file_paths.md

```diff
@@ -292,6 +294,7 @@ create a `puppet` user or group.
             puppet *                          # ruby bin wrapper
         lib *
             puppet.rb *
+        locale *
```

Comment:
So on Windows we have the siblings:
* `bin`
* `lib`
* `locale`

But on the *nix variant above, we have siblings like:

* `bin`
* `lib`
* `share`
  * `puppet`
    * `locale`

So the two relative path structures are not the same.

I see references to a top-level `/opt/share/locale` ... so wondering if `/opt/puppetlabs/puppet/locale` is a sufficient starting point? (i.e. what advantage is there to nesting under `share`)?  That's probably a question for @branan